examples/preprocessing/plot_eog_regression.py

```diff
@@ -0,0 +1,76 @@
+"""
+========================
+EOG regression
+========================
+
+Reduce EOG artifacts by regressing the EOG channels onto the rest of the
+signal.
+
+References
+----------
+[1] Croft, R. J., & Barry, R. J. (2000). Removal of ocular artifact from
+the EEG: a review. Clinical Neurophysiology, 30(1), 5-19.
+http://doi.org/10.1016/S0987-7053(00)00055-1
+
+Authors: Marijn van Vliet <w.m.vanvliet@gmail.com>
+
+License: BSD (3-clause)
+"""
+
+import mne
+from mne.datasets import sample
+from mne.preprocessing import eog_regression
+from matplotlib import pyplot as plt
+
+print(__doc__)
+
+data_path = sample.data_path()
+raw_fname = data_path / 'MEG' / 'sample' / 'sample_audvis_filt-0-40_raw.fif'
+
+# Read raw data
+raw = mne.io.Raw(raw_fname, preload=True)
+events = mne.find_events(raw, 'STI 014')
+
+# Bandpass filter the EEG and EOG
+picks = mne.pick_types(raw.info, meg=False, eeg=True, eog=True)
+raw.filter(0.3, 30, method='iir', picks=picks)
```

Comment:
Nowadays you could just do
```suggestion
raw.filter(0.3, 30, method='iir', picks=('eeg', 'eog'))
```